app/panel/components/Summary.jsx

```diff
@@ -312,6 +312,11 @@ class Summary extends React.Component {
 		return user && user.subscriptionsPlus;
 	}
 
+	_isPremiumSubscriber() {
+		const { user } = this.props;
+		return user && user.scopes && user.scopes.includes('subscriptions:premium');
+	}
+
 	_pageHost() {
```

Comment:
Mimicking the `subscriptionsPlus` approach allows us to abstract away this implementation detail here as well

Reply:
The `_isPremiumSubscriber()` now mimics the `_isPlusSubscriber` function